sdk/servicebus/service-bus/samples/README.md

```diff
@@ -17,6 +17,18 @@ npm install @azure/service-bus
 
 Before running any of the samples, update it with the connection string and the queue/topic/subscription names you have noted down above.
 
+## Initializing the samples folder
+
+Copy the samples folder somewhere in your computer, then run `npm
+install` inside of it.
+
+If you want to run just a specific sample, copy that file into a
+separate folder and continue with the steps below. Remember to
+install the library with `npm install @azure/service-bus`. Read the
+sample's source code to verify any other dependency that you might
+need (dependencies will be visible within the first lines of each
+sample file).
```

Comment:
Sorry for the churn, but I am now wondering if we should make any changes to the readme at all.

Our initial problem was that it is not clear that the user needs to copy the samples into a separate folder so that `npm install @azure/service-bus` works.
Our first solution was to update the README

But now you have found the solution to the original problem by adding the package.json file.

So, at this point, the readme file should be fine the way it is, correct?

Reply:
At the most, I am thinking we replace the below line in the `## Install the library` section 

> Run the below in your samples folder to install the npm package for Azure Service Bus library.

to 


> You can either create your own folder where you copy the individual sample files in or you can clone this repository and work directly in the samples folder. In either case, run the below in your samples folder to install the npm package for Azure Service Bus library.

Thoughts?

Reply:
> You can either create your own folder where you copy the individual sample files or you can clone this repository and work directly in the samples folder. In either case, run the below in your samples folder to install the npm package for Azure Service Bus library.

Maybe verbose ?
I don't feel like mentioning both the cases since there is no effect of copying the folder to somewhere else.

----------------------------------

How about something like this ?

> Copy the samples folder, then run `npm install` inside it in order to install the npm package for Azure Service Bus library.

> If you want to use the source code in this repo instead of the published package, update `"@azure/service-bus"` to `"../../src"` in the samples.

Reply:
I don't understand this part:
> update "@azure/service-bus" to "../../src"

If we have the package.json, there's no need for that. Also, if they copy the source outside of the repo, that would make less sense.

What about:

> Copy the samples folder, then run npm install inside it in order to install the npm package for Azure Service Bus library.
>   
> If you want to copy individual sample files and run them outside, remember to install the service-bus package with: `npm install @azure/service-bus`

Thoughts?

Reply:
@HarshaNalluru We are not trying to handle the case of user wanting to use the source code. We want to achieve 2 things
- have the user copy paste the sample to their own folder, install the library in that folder and run the samples from there
- in case the user finds copy pasting multiple files tedious or decides to just clone the repo because it invloves lesser number of steps :), then they should still be able to install the library there and run the samples.

@sadasant My concern with "Copy the samples folder" is that, as far as I know, one cant really copy a folder without cloning the repo first :) where as one can copy the file contents from the browser or download the individual file.

If the user has already cloned the repo, then they might as well just work from there :) So "copy the folder" is just an extra step

Reply:
@ramya-rao-a One way to simplify this is to add a .zip file either in samples/ or above, and then point to the zip file! The complete thing would look like:

```
wget https://raw.githubusercontent.com/Azure/azure-sdk-for-js/master/sdk/servicebus/service-bus/samples.zip
tar -xvf samples.zip
cd samples
npm i
ts-node typescript/gettingStarted/browseMessages.ts
```

Is that better?

Reply:
- Adding a .zip file would mean that we need to automate it's generation whenever the samples are updated. Should be do-able and Eng Sys team might be able to help.
- In case we add a .zip file, I'd suggest adding a link in the readme itself to directly download the .zip file rather than expecting users to have `wget` installed.

Reply:
For the .zip approach, please discuss with the Engineering Systems team and log an issue if everyone agrees with this approach.

For now, I have updated the text and we can go with what is there in this PR.

Reply:
I made this issue to keep track of the zip idea: https://github.com/Azure/azure-sdk-for-js/issues/3367

Reply:
@HarshaNalluru made an issue before me, so please look at that one instead: https://github.com/Azure/azure-sdk-for-js/issues/3366